### ISSUE TITLE
textarea element 

### ISSUE THREAD

**Issue by neuropass** [albertogasparin/Optiscroll#40]
Hi, does this support <textarea> elements? Thanks!Q

**Comment by albertogasparin** [comment]
No, unfortunately there is no way to customise scrollbars of `textarea` elements directly

**Comment by neuropass** [comment]
Hi, are you sure? this plugin has the option:
https://mobius1.github.io/MiniBar/

Could you look into it and see if you can add the same option? I think its worth it considering your plugin is one of the best and lightest out there. Please :)

**Comment by albertogasparin** [comment]
Thanks for providing an example. That lib uses an interesting trick to solve the limitation. Unfortunately don't have time right now to add support for it, but I'll reopen the issue so it won't get lost.

**Comment by neuropass** [comment]
Hi Alberto, any chance you would have some free time to try to implement this new feature? I really love this scrollbar! I would like to use it for textareas too.
